### DISCUSSION TITLE
When will be the v117 release?

### DISCUSSION THREAD

**Discussion by RoyRao2333** [Alex313031/Thorium-MacOS#10] [General]
I'd like a v117 release for macOS because of the refactored UI for the debugger:

<img width="684" alt="image" src="https://github.com/Alex313031/Thorium-MacOS/assets/31413093/ea5da133-487d-4d8f-8929-a4ee9210ef4c">

**Comment by gz83** [comment]
Version 117 is being released gradually, just wait patiently.

@RoyRao2333

**Reply by RoyRao2333** [reply]
Sure, thanks for the reply!
